spec/statement.md

```diff
@@ -0,0 +1,131 @@
+# Statement
+
+## Table of Contents
+
+* [Definitions](#definitions)
+  * [Current indentation level](#current-indentation-level)
+  * [Document indentation character](#document-indentation-character)
+* [Syntax](#syntax)
+  * [ScssStatements](#scssstatements)
+  * [IndentedStatements](#indentedstatements)
+  * [Statements](#statements)
+  * [Block](#block)
+  * [Whitespace](#whitespace)
+  * [Indentation](#indentation)
+
+## Definitions
+
+### Current indentation level
+
+The `current indentation level` is the count of the [`documentation indentation
+character`] between a new line and any non-whitespace characters of the last
+consumed line.
+
+A document parsed in the indented syntax may not begin with whitespace, and the
+initial `current indentation level` for any document is `0`.
+
+Changes in the indentation level are used by the indented syntax to start and
+end blocks of statements.
+
+[`documentation indentation character`]: #document-indentation-character
+
+### Document indentation character
+
+The `document indentation character` is the character used for calculating the
+[`current indentation level`], and is either a space or tab character.
+
+[`current indentation level`]: #current-indentation-level
+
+It is defined as the first tab or space character used as an [IndentMore]
+production in a document.
+
+In the indented syntax, no character other than the `document indentation
+character` may be used for indentation.
+
+## Syntax
+
+### ScssStatements
+
+<x><pre>
+**ScssStatements**      ::= (Statement ';'?¹)*Statement?
+</pre></x>
+
+1: This production is mandatory unless the previous `Statement` is a
+`LoudComment` or `SilentComment`, or after the final production in a
+`ScssStatements` production, which is either at the end of a `Block` or the end
+of the document.
```

Comment:
You don't need to explicitly exempt the final statement—that's why the BNF ends in `Statement?` without a semicolon. You *do* need to explicitly exempt `';` after a `Statement` that ends in a `Block`, or else `a {b {} c: d}` would be invalid.